design/scheduled-update-check.md

```diff
@@ -0,0 +1,66 @@
+# Scheduled Update Checks for Online Installs
+
+KOTS (Admin Console) does not automatically check for new updates. This proposal adds the 
+ability for KOTS (Admin Console) to check for updates every X period of time for online installations.
+
+## Goals
+
+- Pull new updates automatically for KOTS (Admin Console) online installations
+
+## Non Goals
+
+- Deployment of the new updates
+- Scheduled Update Checks for Airgap Installations
+
+## Background
+
+Many customers assume that scheduled update checks are a given and are surprised when they do not receive 
+updates automatically.
+
+## High-Level Design
+
+1. This proposal is only for online installations in KOTS Admin console.
+2. The default period of time (X) is configurable.
+3. The ability to disable this feature completely.
```

Comment:
:+1: this will be a nice to have.  By default, it's enabled?

Reply:
yes, i'll update to mention that here as well.